src/burnchains/burnchain.rs

```diff
@@ -877,7 +864,28 @@ impl Burnchain {
             &block.block_hash()
         );
 
-        let _blockstack_txs = burnchain_db.store_new_burnchain_block(burnchain, &block)?;
+        burnchain_db.store_new_burnchain_block(burnchain, indexer, &block)?;
+        let block_height = block.block_height();
+
+        let this_reward_cycle = burnchain
+            .block_height_to_reward_cycle(block_height)
+            .unwrap_or(0);
+
+        let prev_reward_cycle = burnchain
+            .block_height_to_reward_cycle(block_height.saturating_sub(1))
+            .unwrap_or(0);
+
+        if this_reward_cycle != prev_reward_cycle {
+            // at reward cycle boundary
+            debug!(
+                "Update PoX affirmation maps for reward cycle {} ({}) block {} cycle-length {}",
```

Comment:
This could be an `info` log -- it only happens once every 2100 blocks

Reply:
Done